config/routes.rb

```diff
@@ -64,4 +64,8 @@
     get '/users/edit' => 'devise/registrations#edit', as: 'edit_user_registration'
     put '/users' => 'devise/registrations#update', as: 'registration'
   end
+
+  match '/404' => 'errors#error_404', via: [:get, :post, :put, :patch]
```

Comment:
You can see these pages for yourself by hitting localhost:3000/400, localhost:3000/422, and localhost:3000/500